File: Serverside/User/Api/Dtos/UserDto.cs
using System;

namespace FNO.User.UserApi.Dtos
{
    public class UserDto
    {
        public Guid Id { get; set; }
        public string Username { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Address { get; set; }
        public string Town { get; set; }
        public int ZipCode { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public string ImageUrl { get; set; }
        public DateTime? CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
    }
}

File: Clientside/Admin/src/models/UserDto.ts
interface UserDto {
    id: string,
    username: string,
    firstName: string
    lastName: string
    address: string
    town: string
    zipCode: number
    email: string
    phone: string
    imageUrl: string
    createdAt: Date
    updatedAt: Date
}


export default UserDto